package.json

```diff
@@ -1,99 +1,106 @@
 {
```

Comment:
All I did here is run fixpack, update some dependencies, and add moment-timezone and fuse.js.